Makefile

```diff
@@ -91,4 +91,3 @@ appimage: build
 	mv etc/packaging/appimages/*.AppImage* etc/packaging/appimages/deploy/
 	chmod 755 etc/packaging/appimages/deploy/*.AppImage
 
-include *.make
```

Comment:
Just curious why we needed to delete this?

Reply:
this originally was used to import the `canon.make` file. Now that the file no longer exists, the Makefile will throw an error because there are no targets to include

Reply:
ah, there was only one *.make file. thank you!